src/main/QafooLabs/Profiler/CurlBackend.php

```diff
@@ -7,6 +7,7 @@ class CurlBackend implements Backend
     private $certificationFile;
     private $connectionTimeout;
     private $timeout;
+    private $proxy;
```

Comment:
That's how I did it first. Right now, the class does not have any global state dependencies, so I was afraid off adding one. The other problem is, that this env variable is probably not available. For instance on Ubuntu, the proxy env variables are not passed to the Apache process by default.

But if you want me to use the env var here, I'll do that.